Views/FretBoard.xaml.cs

```diff
@@ -1,43 +1,164 @@
 namespace Fretty.Views;
-
-public partial class FretBoard : ContentPage
+using Microsoft.Maui.Controls.Shapes;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Graphics;
+using System;
+using Microsoft.Maui;
+using System.Collections.Generic;
+public partial class FretBoard: IFretBoard
 {
-    private double _startX;
-    private double _startY;
-    
+    private const int NumRows = 7;
```

Comment:
These probably make more sense as a constructor parameter, and passed into corresponding `Generate..()` functions.

Reply:
she been updated